exercises/tournament/example.rs

```diff
@@ -30,12 +27,9 @@ impl TeamResult {
     }
 }
 
-pub fn tally(input_filename: &Path, output_filename: &Path) -> Result<u32> {
-    let reader = BufReader::with_capacity(2048, File::open(input_filename).unwrap());
-    let mut count = 0;
+pub fn tally(input: &str) -> String {
```

Comment:
I would like a reminde ron the idiomatic-ness of `str` and `String`. I will see if I can find an answer myself.